src/Components/ProgressBar.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+type ProgressProps = {
+  questionsLength :number;
+  answersLength: number;
+};
+let ProgressBar = (props :ProgressProps) =>{
+  let percentage = Math.ceil(props.answersLength / props.questionsLength *100)
+ return (
+   <section>
+   <section style={{width : percentage, backgroundColor:'red'}}>
```

Comment:
Need to create Inline style because it is dynamic. However we need to style the container to hold progress bar correctly.